src/main.rs

```diff
@@ -27,44 +27,55 @@ async fn main() -> anyhow::Result<()> {
     };
     let cfg = proj_dirs.config_dir().join("websocket-token");
 
-    let exists = tokio::fs::try_exists(&cfg).await;
-    if !exists.unwrap_or(false) {
-        anyhow::bail!(
-            "\
-No OBS WebSocket authentication token found.
-Open OBS, go to Tools -> WebSocket Server Settings, \
-hit 'Show Connect Info', and place that token in the file {}\
-            ",
-            cfg.display()
-        );
-    }
-    let pw = tokio::fs::read_to_string(&cfg)
-        .await
-        .context("read OBS WebSocket autheticator from obs-do config file")?;
+    let exists: Result<bool, std::io::Error> = tokio::fs::try_exists(&cfg).await;
 
-    let client = if let Ok(client) = Client::connect("localhost", 4455, Some(pw.trim())).await {
-        let version = client
-            .general()
-            .version()
-            .await
-            .context("get OBS version")?;
-        eprintln!(
-            "Connected to OBS: {} / {}",
-            version.obs_version, version.obs_web_socket_version
-        );
-        client
+    let pw = if exists.unwrap_or(false) {
```

Comment:
I think we should probably start handling the case where `exists` is `Err(e)` directly and bail out with an error message.

Reply:
Is this better now ?
We are Rust beginners ;p

Reply:
I think you're doing well for being a beginner! Left a suggestion for a slightly nicer way to do it :+1: